src/libraries/Microsoft.Extensions.Configuration.Binder/tests/ConfigurationBinderTests.cs

```diff
@@ -994,6 +1040,84 @@ public void CanBindValueTypeOptions()
             Assert.Equal("hello world", options.MyString);
         }
 
+        [Fact]
+        public void CanBindImmutableClass()
```

Comment:
Any tests with nested immutable objects?

Reply:
Good point - forgot to add that previously. Added now.